test/draw.py

```diff
@@ -56,49 +33,68 @@ class TestDraw(unittest.TestCase):
 
     # Result image for draw
     def test_draw(self):
-        image = Image.new(imageMode, imageSize, imageBackground)
-        canvas = ImageDraw.Draw(image)
 
-        for i in range(len(points)):
-            draw(canvas, (points[0], points[1]), 'A')
+        """ Create the file name and its saving path, and specify the reference file to compare to."""
```

Comment:
function help string format should have the """ separator alone on it's own line:

"""
This function ...
"""